efs/efs.tf

```diff
@@ -0,0 +1,16 @@
+# EFS File System
+
+resource "aws_efs_file_system" "project_efs" {
+  creation_token = "${var.project_name}-file-system"
+  throughput_mode = "bursting"
+  performance_mode = "generalPurpose"
+  encrypted = false
+}
+
+# EFS Moutn Target
+
+resource "aws_efs_mount_target" "project_mount_target" {
+  file_system_id = aws_efs_file_system.project_efs.id
+  subnet_id      = var.private_subnet_id
+  security_groups = [var.security_groups[0].id]
```

Comment:
instead of using a variable here please add to your code a sg that will allow efs to be accessed from ECS nodes

- resource "aws_security_group"
- resource "aws_security_group_rule"
  - type ingress
  - protocol tcp
  - to/from port 2049 
  - sg id security group created above
  - source sg id use a input variable called `var.ecs_nodes_sg_id`